src/components/common/SendCode/layouts/CodeRequested.vue

```diff
@@ -0,0 +1,26 @@
+<template>
```

Comment:
maybe rename this file to `CountLabel` or something else for showing the `state` in component name, but not the `action` or `what was`